src/cli/client_manager.py

```diff
@@ -24,8 +23,6 @@ def __init__(self, node_endpoint, signer_endpoint) -> None:
             else:
                 self.node_endpoint = 'https://' + self.node_endpoint
                 logger.info("Node endpoint URL points to an SSL endpoint. Using HTTPS protocol prefix.")
-        if len(self.node_endpoint.split(':')) < 3:
-            self.node_endpoint += f':{TEZOS_RPC_PORT}'
```

Comment:
Why did you remove this?

Reply:
The tezos rpc port url does not work anymore. Thus I removed it.